docs/TSUNAMI_METADATA_CONVENTIONS.md

```diff
@@ -0,0 +1,31 @@
+# Tsunami Metadata Conventions
+
+In this document we describe the metadata conventions used for GeoNet sensors that monitor tsunami.
+
+## Overview
+
+GeoNet has two types of tsunami monitoring stations: tide gauges and DART buoys. The distinction is made by the primary purpose of the station; whether it exists principally to record tsunami arrivals on-shore (via a tide gauge) or in the deep ocean (via a DART buoy).
+
+The management of DART data and metadata is an evolving space, and until this is more established we refrain from documenting it here.
+
+Tide gauge stations operated by GeoNet are not measuring absolute sea level, as they are not fully calibrated to the mean sea level. Data obtained by GeoNet tide gauges are measuring water height above the sensor.
+
+Tide gauge data and metadata mostly follow the conventions of seismic data and metadata, and unless stated otherwise in this document these conventions are what apply to tide gauge data and metadata. The sibling document `SEISMIC_METADATA_CONVENTIONS.md` details these conventions. Here we make reference only to the differences between the tide gauge metadata conventions and those of seismic metadata.
+
+## Location Code
+
+For tide gauges, the location code indicates primary and backup sensors at the station. For example, a station may have sites at location codes `41` and `42`, and these location code are used to distinguish the respective data streams from the sensors at each site. In this way, we can think of the location code for tide gauges as a "sensor code". 
```

Comment:
Might be worth changing this to `40` and `41` as this is typically the majority of tsunami gauge location codes used (I'm not actually if we have one with `42` and if they were to be moved it would as a pair so would become `42` and `43`), the majority of DART buoys also use `40` as a starting point with one re-deployed as `41`.